src/shared/components/StyledSlider/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { styled } from '@mui/material/styles'
+import Slider from '@mui/material/Slider'
+
+export const StyledSlider = styled(Slider)(({ theme }) => ({
```

Comment:
This whole stylesheet looks shady, like you don't need it, please figure out what is needed here and what can be deleted and then do it 🤌🏼